package.json

```diff
@@ -722,6 +722,21 @@
           "markdownDescription": "`#editor.lineNumbers#` is determined by the active Vim mode, absolute when in insert mode, relative otherwise.",
           "default": false
         },
+        "vim.smartQuotes.enable": {
+          "type": "boolean",
+          "markdownDescription": "Enable smart quotes movements (similar to targets.vim)",
```

Comment:
Maybe make `targets.vim` a link to the original plugin? ([this anchor](https://github.com/wellle/targets.vim#quote-text-objects) seems like the equivalent)

Reply:
```suggestion
          "markdownDescription": "Enable smart quotes movements (similar to [targets.vim](https://github.com/wellle/targets.vim#quote-text-objects))",
```